src/components/Chart/DonutChart.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import C3Chart from 'react-c3js';
+import { findDOMNode } from 'react-dom';
+import { lifecycle, compose } from 'recompose';
+import { patternfly, getComposer } from './constants';
+
+const pfSetDonutChartTitle = patternfly.pfSetDonutChartTitle;
+
+const addDonutTitle = lifecycle({
+  componentDidMount() {
```

Comment:
two comments:
1. should this be only on mount, maybe on update as well?
2. maybe it make sense to test the props before calling the pf function (as i assume it would be faster not to call it if its not needed).

Reply:
👍 Thanks.

Set also the 'componentDidUpdate' method.
Testing for change is a little tricky and may take the same time as re draw ... need to think how to do that ...